ibft/sync/history_test.go

```diff
@@ -281,63 +281,6 @@ func TestFindHighest(t *testing.T) {
 			10,
 			"",
 		},
-		//
```

Comment:
@amir-blox did we cover all scenarios in the tests?

Reply:
The following scenarios are covered:
1. valid
2. empty responses
3. error responses
4. some errors, some valid
5. valid multi responses
6. valid multi responses different seq

I've added another scenario - "some errors, some empty"
Cn you think of other scenarios besides that?

Reply:
Added scenarios:
* all invalid
* some invalid, some empty